packages/user-operation-controller/src/types.ts

```diff
@@ -315,10 +315,10 @@ export type SmartContractAccount = {
  */
 export type UserOperationReceipt = {
   /** Confirmed total cost of the gas for the user operation. */
-  actualGasCost: string;
+  actualGasCost: string | number;
```

Comment:
If we're normalizing these values, we should only ever be persisting them in the same format, regardless of the response.

So just `string` in our case, or even `Hex` for clarity.